.github/workflows/CI.yml

```
@@ -7,14 +7,19 @@ env:
 jobs:
   build:
 
-    runs-on: macOS-15
+    runs-on: macOS-26
 
     steps:
     - uses: actions/checkout@v5
 
-    - uses: maxim-lobanov/setup-xcode@v1.6.0
+    - name: Read Xcode version
+        id: xcode_version
```

actionlint (1.7.8)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)